packages/core/src/semantic-icon-provider/SemanticIconProvider.tsx

```diff
@@ -0,0 +1,90 @@
+import {
+  CalendarIcon,
+  ChevronDownIcon,
+  ChevronLeftIcon,
+  ChevronRightIcon,
+  ChevronUpIcon,
+  CloseIcon,
+  ErrorSolidIcon,
+  InfoSolidIcon,
+  OverflowMenuIcon,
+  StepActiveIcon,
+  StepDefaultIcon,
+  StepSuccessIcon,
+  SuccessTickIcon,
+  TearOutIcon,
+  UserSolidIcon,
+  WarningSolidIcon,
+} from "@salt-ds/icons";
+import {
+  type ElementType,
+  type ReactNode,
+  createContext,
+  useContext,
+} from "react";
+
+export type SemanticIconMap = {
+  DownIcon: ElementType;
+  UpIcon: ElementType;
```

Comment:
is this weird ?
I was expecting semantics like, forward and back and expand and collapse ?

Reply:
this is in sync with the file Paul had provided: https://www.figma.com/design/xET9vAEtgJ7w29lGNGAcrF/Salt-(Next)-Iconography?node-id=2230-1418&node-type=canvas&m=dev